### DISCUSSION TITLE
Hyperlinks [Jul 23]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2901] [Discord Threads]
> 🙋‍♂️ **Kelsey**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@962537363983196240> No. Storing hyperlinks on text-columns is not supported. You can store links themselves in columns of type `URL`.  
![image0](https://cdn.discordapp.com/attachments/999760781824491570/999893375274725386/Screenshot_2022-07-22_at_9.48.54_AM.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999760781824491570)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
